sheepdog/transactions/upload/sub_entities.py

```diff
@@ -709,8 +709,8 @@ def _get_file_size(self):
             return self.doc.get("file_size")
         return None
 
-    def _create_alias(self, **kwargs):
-        return self.transaction.index_client.create_alias(**kwargs)
+    def _create_alias(self, alias, did):
```

Comment:
does this function have any tests?

Reply:
A lot of the integration tests touch it, I'm not certain about unit tests. Let me check!

Reply:
Update: The function is tested by integration tests only, it looks like the bulk of sheepdog's tests are integration tests.

Reply:
a lot of unit tests make sure the `_create_alias` function is called when it should. they don't test the result, but all this function does is call the indexclient function, which has unit tests in the indexclient repo